### ISSUE TITLE
Support Emacs 27.1

### ISSUE THREAD

**Issue by doolio** [Trevoke/org-gtd.el#147]
Emacs 27.1 is still the version on Debian Stable. Would you consider still supporting it?

Edit: I see now from your info manual it was proving too difficult for you to maintain support. Have you heard of the [compat](https://github.com/emacs-compat/compat) library?

Aside: If updating the docs, is it necessary to update both the .texi and .org file? I noted one or two typos and can submit a PR for consideration.

**Comment by Trevoke** [comment]
I would, although doing manual testing on the package across three
different versions of emacs starts to get pretty heavy for just me.

I ran the tests locally (I am currently on Ubuntu and I also have 27.1) and
they pass. I am also able to set up continuous integration to run on 27.1.

My problem is primarily that the tool I use for project management (eldev)
relies on silex/emacs docker images to create stable environments for
testing purposes, and that doesn't provide a tag for emacs 27.1 yet. I just
opened an issue for it: https://github.com/Silex/docker-emacs/issues/94 - I
think when that tag gets added, if someone is willing to run the tool
through some beta/usability testing (either me when I have time next, or a
user), then I can bring support for 27.1 back.

I think I've figured out most of the actual compatibility problems I had,
but, again, I need to be able to run the automated tests when I do
development, and this at the moment is not something I can easily guarantee
I can do, which means I cannot claim I suppose 27.1.
